src/history/index.js

```diff
@@ -38,3 +69,7 @@ export async function publish() {
     versionRecorder.publish()
   ]);
 }
+
+export function getLastSnapshot(serviceId, documentType) {
```

Comment:
Let's use consistent adjectives 🙂 

```suggestion
export function getLatestSnapshot(serviceId, documentType) {
```